en/sql/tuning.rst

```diff
@@ -3829,6 +3829,157 @@ The join with the *right_tbl* table was eliminated.
+
+#. The view includes a **DISTINCT** clause.
+
+#. **CTE** (Common Table Expressions) is included in the query.
+
+#. Using **OUTER JOIN** with a view.
+
+#. Using aggregation or analytical functions.
+
+#. Using **ROWNUM, LIMIT**, or **GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM()**.
+
+#. Written using **Correlated Subquery**.
+
+#. The view includes methods.
+
+#. The view includes **RANDOM(), DRANDOM(), SYS_GUID()**.
+
+The following is an example that includes the **CONNECT BY** clause.
+
+.. code-block:: sql
+
+        -- Creating a tree table and then inserting data
+        CREATE TABLE tree(ID INT, MgrID INT, Name VARCHAR(32), BirthYear INT);
+
+        INSERT INTO tree VALUES (1,NULL,'Kim', 1963);
+        INSERT INTO tree VALUES (2,NULL,'Moy', 1958);
+        INSERT INTO tree VALUES (3,1,'Jonas', 1976);
+        INSERT INTO tree VALUES (4,1,'Smith', 1974);
+        INSERT INTO tree VALUES (5,2,'Verma', 1973);
+        INSERT INTO tree VALUES (6,2,'Foster', 1972);
+        INSERT INTO tree VALUES (7,6,'Brown', 1981);
+
+        -- Executing a hierarchical query with the CONNECT BY clause
+        SELECT *
+        FROM (SELECT * FROM tree WHERE BirthYear = 1973) t
+        CONNECT BY PRIOR t.id=t.mgrid;
+
+Due to the use of the **CONNECT BY** clause in the above query, **View Merging** cannot be performed.
+
+The following is an example where a view includes the **DISTINCT** clause.
+
+.. code-block:: sql
+
+        SELECT * FROM (SELECT DISTINCT host_year FROM record) T;
+
+Due to the **DISTINCT** clause used within the view in the above query, View Merging cannot be performed.
+
+The following is an example that includes **CTE** (Common Table Expressions) in the query.
+
+.. code-block:: sql
+
+        WITH cte AS (SELECT * FROM athlete WHERE gender = 'M') 
+        SELECT * FROM cte WHERE cte.nation_code = 'USA';
+
+Queries that contain **CTE** like the above cannot undergo View Merging.
+
+The following is an example performing an **OUTER JOIN** with a view.
+
+.. code-block:: sql
+
+        SELECT * 
+        FROM athlete a 
+        LEFT OUTER JOIN (SELECT * FROM record WHERE host_year = 2020) b 
+        ON a.code = b.athlete_code;
+
+In cases where an **OUTER JOIN** is performed as above, View Merging cannot be executed.
+
+The following is an example using aggregate or analytical functions.
+
+.. code-block:: sql
+
+        SELECT * 
+        FROM (SELECT AVG(host_year) FROM record WHERE medal = 'G') a;
+
+Queries that include aggregate or analytical functions do not qualify for View Merging.
+
+The following is an example using **ROWNUM, LIMIT** or **GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM()**.
+
+.. code-block:: sql
+
+        SELECT *
+        FROM (SELECT gender, rownum FROM athlete WHERE rownum < 15) a
+        WHERE gender = 'M';
+
+Queries that utilize ROWNUM, LIMIT or GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM() cannot undergo View Merging.
+
+The following is an example crafted using a **Correlated Subquery**.
```

Comment:
It is trivial.

In the manual, I think it is important to use simple words and write intuitively.
And I think using similar words for similar sentences helps to maintain consistency.

Reply:
I use 'In cases where Query includes ROWNUM, LIMIT or GROUPBY_NUM(), INST_NUM(), ORDERBY_NUM(), **View Merging** cannot be executed.' instead of `undergo`.